kea-vscode/src/views/pull-request-contents/local-commit/local-folder-tree-node.ts

```diff
@@ -0,0 +1,31 @@
+import * as vscode from "vscode";
+import { CollapsibleState, getCollapsibleState, IParentTreeNode } from "../../tree-node";
+import { LocalFileTreeNode } from "./local-file-tree-node";
+
+/**
+ * Represents a folder in a local commit tree.
+ */
+export class LocalFolderTreeNode implements IParentTreeNode<LocalFileTreeNode | LocalFolderTreeNode> {
+  #contextValue = "localFolder";
+  #iconPath = new vscode.ThemeIcon("folder");
```

Comment:
There should be no icon for folders. Please apply to this other folders in the repo.